compiler/rustc_ast/src/ast.rs

```diff
@@ -2705,8 +2705,19 @@ impl Item {
 /// `extern` qualifier on a function item or function type.
 #[derive(Clone, Copy, Encodable, Decodable, Debug)]
 pub enum Extern {
+    /// No explicit extern keyword was used
+    ///
+    /// E.g. `fn foo() {}`
     None,
+    /// An explicit extern keyword was used, but with implicit ABI
+    /// 
+    /// E.g. `extern fn foo() {}`
+    ///
+    /// On most ABIs, this is just `extern "C"` (see `rustc_target::spec::abi::Abi::FALLBACK`)
```

Comment:
Isn't this always `extern "C"`?

Reply:
Ah, for some reason I thought it was a trait assoc const.